File: Engine/src/main/java/org/etieskrill/engine/graphics/gl/shader/impl/LightSourceShader.java
package org.etieskrill.engine.graphics.gl.shader.impl;

import io.github.etieskrill.injection.extension.shaderreflection.ReflectShader;
import org.etieskrill.engine.graphics.gl.shader.ShaderProgram;

import java.util.List;

@ReflectShader
public class LightSourceShader extends ShaderProgram {
    public LightSourceShader() {
        super(List.of("LightSource.vert", "LightSource.frag"), false);
    }
}

File: Engine/src/commonMain/kotlin/org/etieskrill/engine/graphics/shader/impl/LightSourceShader.kt
package org.etieskrill.engine.graphics.shader.impl

import io.github.etieskrill.injection.extension.shader.reflection.ReflectShader
import org.etieskrill.engine.graphics.shader.Shader

@ReflectShader
class LightSourceShader : Shader(listOf("shaders/LightSource.vert", "shaders/LightSource.frag"), false)
